frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -1031,6 +1036,7 @@ export const widgets = [
         visibility: { value: '{{true}}' },
         disabledState: { value: '{{false}}' },
         borderRadius: { value: '{{0}}' },
+        textColor: { value: '' },
```

Comment:
Can you use the current input color as the default value?
```suggestion
        textColor: { value: '#232e3c' },
```